### ISSUE TITLE
[NPC] Evergrove Druid - messes with some quests

### ISSUE THREAD

**Issue by mhdemn** [Atlantiss/NetherwingBugtracker#2524]
**Description**: These [NPC](https://db.excalibur.ws/?npc=22423)s are all over Death's Door in BEM. First quest that sends you to this place, called [Death's Door](https://db.excalibur.ws/?quest=10910), also gives you a [flare thing](https://db.excalibur.ws/?item=31763) to call down some of the druids flying above in bird form, since either of them is your questgiver.

**Current behaviour**: When called upon it will fly down to the ground then fly back up instantly, giving you no time to turn in and accept follow-up quests.

Edit: Actually they shapeshift super briefly into humanoid form and then fly back up, but do this so fast it will bug and when flying back up they won't have a form.

![image](https://user-images.githubusercontent.com/25901582/48301805-e978e400-e4f3-11e8-8184-ff32edc79940.png)
The guy's far above but you can see in the portrait he's bodyless and there's no shape at all against the mountain-spike thing.

This will make it nearly impossible without some addon like Vanilla's LazyPig (a kind of addon I haven't found yet, at least in a working version) to deliver and accept following quests with such a small time window to interact. The following quests are affected:

[Death's Door](https://db.excalibur.ws/?quest=10910)
[Fire At Will!](https://db.excalibur.ws/?quest=10911)
[Harvesting the Fel Ammunition](https://db.excalibur.ws/?quest=10904)

**Expected behaviour**: 
- When called upon it should:

  - Fly down,
  - Shapeshift into humanoid form,
  - Remain for a short while in the ground (say 1-2 minutes, can't remember with exactitude nor find retail TBC footage).

In this [retail video](https://www.youtube.com/watch?v=kGDhHXSHhds) though you can see it's behaviour. The part where the NPC seeks the player changing route in real time, instead of going to the place he was called upon from, might be a behaviour that was implemented in a later xpac.

What I think is clear is listed in the previous points.

**Server Revision**: 2279

**Comment by mhdemn** [comment]
Update: When I completed [Fire at Will!](https://db.excalibur.ws/?quest=10911) the druid that flew down behaved like the one in the retail video (except for the part where he goes to the player, this went to a location a few meters away): I couldn't interact with him until he reached the ground and transformed into a humanoid. When I accepted the follow-up quest [The Hound-Master](https://db.excalibur.ws/?quest=10912) (that I forgot to list in the post) he did a line of speech saying he had to go and despawned.

This seems a normal behaviour, only difference I see here is the location I used the flare item from.

[This](https://youtu.be/vNU7HSTOSGg) (62.8,65.5) is where I used it for every quest except when turning Fire At Will and picking The Hound-Master.

Following pic (62.3,59.3) is where it worked flawlessly:
